src/data/items.h

```diff
@@ -1628,7 +1628,11 @@ const struct Item gItems[] =
     [ITEM_NUGGET] =
     {
         .name = _("Nugget"),
-        .price = 10000,
+        #if I_SELL_VALUE >= GEN_9
+            .price = 20000,
+        #else
+            .price = 10000,
+        #endif
```

Comment:
not as sold on this solution compared to the other, but we could use [constant conditional initalization](https://cexplore.karathan.at/z/VOyba2) for these (and potentially other occurances of preproc statements like that) like so:
```c
    .price = (I_SELL_VALUE >= GEN_9) ? 20000 : 10000,
```

Reply:
Good idea, but I'll leave it for another PR to change the other ones